modules/auth/user_form.go

```diff
@@ -6,6 +6,9 @@ package auth
 
 import (
 	"mime/multipart"
+	"strings"
```

Comment:
Please add Gitea copyright to header.

Reply:
See 
https://github.com/go-gitea/gitea/pull/5157/commits/2f35804137e45ec0c94335e6de18ea1a8fa2724b